cal_coefficients.py

```diff
@@ -6,40 +6,37 @@
 
 """
 import os
-clear = lambda : os.system('cls')
+
+clear = lambda: os.system('cls')
 clear()
-import numpy as np
-import scipy.io as sio
-from receiver_calibration_func import *
+import receiver_calibration_func as rcf
 import glob
-import reflection_coefficient as rc
-from S11_correction import *
+import S11_correction as s11
```

Comment:
calling this as s11 is generating an error. Line 442 is using 's11' as a variable, one or the other will have to change in the next commit.

Reply:
Yup fair point, fixed in 30d389d5952adaa9097c7b9c5876f35d8818b74c